apps/greenhouse/package.json

```diff
@@ -15,6 +15,12 @@
   "private": true,
   "devDependencies": {
     "@cloudoperators/juno-config": "*",
+    "@cloudoperators/juno-app-doop": "*",
+    "@cloudoperators/juno-app-heureka": "*",
+    "@cloudoperators/juno-app-supernova": "*",
+    "@cloudoperators/juno-messages-provider": "*",
+    "@cloudoperators/juno-oauth": "*",
+    "@cloudoperators/juno-ui-components": "*",
```

Comment:
shouldn't these dependencies remain as solid dependencies rather than dev?

Reply:
For applications, dependencies are bundled at build time, meaning all required components become part of the app during the build process. Therefore, post-build, the specific handling of dependencies no longer matters.

Reply:
When you run vite build --mode static, Vite uses dependencies (dependencies in package.json)

Reply:
That would surprise me, but I can move deps back, no problem!

Reply:
Normally when we build an app or package the `NODE_ENV` gets set to `production` and in that case the `devDependencies` are not installed and bundled.